### ISSUE TITLE
Phase_1-demo-EDA_model

### ISSUE THREAD

**Issue by amindh** [ThomasBourgeois/zephyr#1]
Dans la section windspeed, lors de la separation du jeu de données à l'aide de la fonction train_test_split, l'option shuffle est activée (True). Etant donné que nous avons des données temporelles, cela va provoquer un "Data leakage" lors de la separation et mener à des résultats biaisés. Il faut donc mettre shuffle = False.

**Comment by ThomasBourgeois** [comment]
Dans la phase 1 du challenge, nous avons convenu que nous acceptions le data leakage temporel.

Nous n'accepterons pas celui-ci dans la phase 2 en revanche.
